Development/DCM/DCMDatabase/dbpm.py

```diff
@@ -5,15 +5,26 @@
 
 from src.dcm_constants           import *
 from peewee                      import *
-import DCMUserAccountManager.duam  
+#from DCMUserAccountManager.duam  import User 
 
 
 database = SqliteDatabase(C_DATABASE_PATH, pragmas={
     'journal_mode': C_DATABASE_JOURNAL_MODE,
     'cache_size': C_DATABASE_CACHE_SIZE,
     'foreign_keys': C_DATABASE_FOREIGN_KEYS})
 
-
+class User:
+    def __init__(self, p_username, p_password, p_userRole):
+        self.username    = p_username
+        self.password    = p_password
```

Comment:
this shouldn't be here.. this doesn't relate to the database user data, this relates to the user in the user account manager